test/functional/types.js

```diff
@@ -198,6 +199,40 @@ describe('Types', function() {
                 }]);
             });
         });
+        it("return string if numeric data saved in string typed field", function() {
+            return router.request({
+                uri: '/restbase.cassandra.test.local/sys/table/typeTable/',
+                method: 'put',
+                body: {
+                    table: "typeTable",
+                    attributes: {
+                        string: 'num_string_test',
+                        int: 10,
+                        boolean: false,
+                        num_string: '2015'
```

Comment:
Would be better to set it to `'0'` so that we also test the corner case for `if(!value)`